gcp/iam/test_only_allowed_org_accounts.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from gcp.iam.resources import project_iam_bindings
+
+
+@pytest.fixture
+def allowed_org_domains(pytestconfig):
+    return pytestconfig.custom_config.gcp.allowed_org_domains
+
+
+EXCLUDED_ROLES = ["roles/logging.viewer"]
```

Comment:
might want to add this to the config at some point

Reply:
agreed, will most likely be following up with an addition or two to this list, will make the change then.